DevelopGitHub/GitHubExtension/GitHubExtention.Preferences.WebApi/Azure/AzureBlobStorageMultipartProvider .cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using GitHubExtention.Preferences.WebApi.Queries;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace GitHubExtention.Preferences.WebApi
+{
+    public class AzureBlobStorageMultipartProvider : MultipartFileStreamProvider
```

Comment:
If I use interface instead of AzureBlobStorageMultipartProvider. I can't use ReadAsMultipartAsync<AzureBlobStorageMultipartProvider>(_multipartStreamProvider) becuase this method requires paramater refernces to abstract class MultipartStreamProvider. As a result I need to make my own conten reading provider. Am I understand you?